docs/src/main/paradox/index.md

```diff
@@ -1,273 +1,93 @@
-# Akka Distributed Workers
+# Akka Distributed Workers with Scala Guide
+ 
+Akka is a toolkit and runtime for building highly concurrent, distributed, and fault-tolerant event-driven applications on the JVM. Akka can be used with both Java and Scala.
+This guide introduces Akka by describing the Scala version of the Hello World example. If you prefer to use Akka with Java, switch to the [Akka Distributed Workers with Java guide](http://developer.lightbend.com/guides/akka-quickstart-java/).
 
-##The Goal
+This guide demonstrates more advanced usage of Akka and requires familiarity with Actors. If you have no previous experience with Actors you should start with [Akka Quickstart with Scala](http://developer.lightbend.com/guides/akka-quickstart-scala/) which goes through the basics.
 
-Some applications need to distribute work to many machines because one single box
-obviously has limited resources. This tutorial will show one way to implement
-distributed workers using Akka cluster features.
+Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed. This will get you started building distributed systems with Akka.
 
```

Comment:
I suggest adding a level 2 heading "Example overview" just before the download instructions and then adding something like:

"To be reactive, distributed applications must deal gracefully with temporary and prolonged outages as well as have the ability to scale up and down to make the best use of resources. Akka clustering provides these capabilities so that you don't have to implement them yourself. The distributed workers example demonstrates the following Akka clustering capabilities (this is just a slightly reworded version of the more technical bulleted list below that explains the app requirements):

- elastic addition and removal of the front-end actors that accept client requests
- elastic addition and removal of the back-end actors that perform the work
- distribution of actors across different nodes
- how jobs are re-tried in the face of failures

But before we dive into how the example accomplishes these goals, download the example and try it out!